common/tables/table-en.html

```diff
@@ -0,0 +1,520 @@
+---
+{
+	"title": "GCWeb Table example",
+	"language": "en",
+	"altLangPage": "table-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-02-28"
+}
+---
+
+<h2 id="tables-example" class="wb-prettify">Basic example</h2>
+<p>For basic styling&mdash;light padding and only horizontal dividers&mdash;add the base class <code>.table</code> to any <code>&lt;table&gt;</code>. It may seem super redundant, but given the widespread use of tables for other plugins like calendars and date pickers, we've opted to isolate our custom table styles.</p>
+<div class="bs-example" data-example-id="simple-table">
+	<table class="table">
+		<caption>Optional table caption.</caption>
+		<thead>
+			<tr>
+				<th>#</th>
+				<th>First Name</th>
+				<th>Last Name</th>
+				<th>Username</th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr>
+				<th scope="row">1</th>
+				<td>Mark</td>
+				<td>Otto</td>
+				<td>@mdo</td>
+			</tr>
+			<tr>
+				<th scope="row">2</th>
+				<td>Jacob</td>
+				<td>Thornton</td>
+				<td>@fat</td>
+			</tr>
+			<tr>
+				<th scope="row">3</th>
+				<td>Larry</td>
+				<td>the Bird</td>
+				<td>@twitter</td>
+			</tr>
+		</tbody>
+	</table>
+</div>
+<!-- /example -->
+<pre><code class="language-html" data-lang="html"><span class="nt">&lt;table</span> <span class="na">class=</span><span class="s">"table"</span><span class="nt">&gt;</span>
+	...
+<span class="nt">&lt;/table&gt;</span></code></pre>
```

Comment:
Remove the span and use the [prettify plugin ](https://wet-boew.github.io/wet-boew/demos/prettify/prettify-en.html)to enhance the code instead

```suggestion
<pre><code>&lt;table class="table"&gt;
	...
&lt;/table&gt;</code></pre>
```